### DISCUSSION TITLE
[A-L] (2023/24) Foglio 9 - Esercizio 2

### DISCUSSION THREAD

**Discussion by Elia-Belli** [sapienzastudentsnetwork/calcolo-delle-probabilita#194] [Esercizi - Bertini] [answered]
![Screenshot 2023-11-27 134903](https://github.com/sapienzastudentsnetwork/calcprob2324/assets/107944476/15ab4f07-a36f-4475-b00a-4088eb08a016)

**Comment by CiottoloMaggico** [comment]
![image](https://github.com/sapienzastudentsnetwork/calcprob2324/assets/79638284/a7096208-343e-4eaf-b5f0-6ed285ed3f23)

**Comment by Elia-Belli** [comment, marked as answer]
## Soluzione confermata in classe
Indichiamo con $f_X,f_Y$, rispettivamente, le densità di probabilità di $X_1,X_2$, che essendo v.a. uniformi identitcamente distribuite in $[0,1]$ assumono i valori:

$$f_Y(x)=f_X(x)=\begin{cases}0\ \ \ \ \ x>1\\
1\ \ \ \ \ 0\leq x\leq 1\\
0\ \ \ \ \ x <0\end{cases}$$

**(1)** Ricordiamo la distribuzione della somma $Z=X+Y$ di v.a. discrete indipendenti:
 $$\mathbb{P}(Z=z)=\sum_{x\in Im(X)}\mathbb{P}(X=x)\mathbb{P}(Y=z-x)$$
Approssimando l'integrale con la somma di Riemann otteniamo:
$$f_Z(z)=\int_{-\infty}^{+\infty}f_X(x)f_Y(z-x)\ dx$$

$Z$ può assumere solo valori nell'intervallo $[0,2]$, quindi la sua densità è nulla per tutti gli altri valori.
Inoltre $f_Y$ è non nulla solo se $(z-x)\in [0,1]$, allora dobbiamo considerare le casistiche $z\in [0,1],z\in [1,2]$ separatamente, rispettivamente:

$$\int_0^z1\ dx=z\ \ \ \ \ \int^2_z1\ dx=2-z$$

$$f_Z(z)=\begin{cases}0\ \ \ \ \ \ \ \ \  \ \ \ \ z<0\\
z\ \ \ \ \ \ \ \ \ \ \ \ \ 0\leq z\leq 1\\
(2-z)\ \ \ 1< z\leq 2\\
0\ \ \ \ \ \ \ \ \ \ \ \ \  z >2\end{cases}$$

**(2)** $Z=max(X_1,X_2)$ assume valori in $[0,1]$, calcoliamo la distribuzione di probabilità $F_Z$ per ottenere la densità $f_Z$, in particolare per il massimo di v.a. continue indipendenti possiamo dire che
$$F_Z(z)=\mathbb{P}(max(X_1,X_2) \leq z)=\mathbb{P}(X_1 \leq z,X_2\leq z)=\mathbb{P}(X_1\leq z)\mathbb{P}(X_2\leq z)=$$

$$=\mathbb{P}(x_1\in[0,z])\mathbb{P}(x_2\in [0,z])=\int^z_0f_X(x)\ dx \cdot \int^z_0f_Y(x)\ dx=\int^z_01\ dx \cdot \int^z_01\ dx=z\cdot z=z^2$$

Sappiamo però che $f_Z$ è la primitiva di $F_Z$:
$$f_Z(z)=\frac{d}{dz}F_Z(z)\Rightarrow f_Z(z)=2z\ \ \ \ z\in[0,1]$$

> Come verifica vediamo se $f_Z$ rispetta la condizione di normalizzazione: $\int^{+\infty}_{-\infty}2z\ dz=\int_0^12z\ dz=z^2|^1_0=1$

**(3)** $Z=min(X_1,X_2)$ assume valori in $[0,1]$, calcoliamo la distribuzione di probabilità $F_Z$ per ottenere la densità $f_Z$:
$$F_Z(z)=\mathbb{P}(min(X_1,X_2)\leq z)=1-\mathbb{P}(min(X_1,X_2)> z)=1-[\mathbb{P}(X_1>z)\cdot \mathbb{P}(X_2)>z)]=$$

$$=1-[\mathbb{P}(x_1\in(z,1])\cdot \mathbb{P}(x_2\in (z,1])]=1-[\int^1_z1\ dx\cdot \int^1_z1\ dx]=1-(1-z)^2=2z-z^2$$

Sappiamo però che $f_Z$ è la primitiva di $F_Z$:
$$f_Z(z)=\frac{d}{dz}F_Z(z)\Rightarrow f_Z(z)=2-2z\ \ \ \ z\in[0,1]$$

> Come verifica vediamo se $f_Z$ rispetta la condizione di normalizzazione: $\int^{+\infty}_{-\infty}2-2z\ dz=\int_0^12-2z\ dz=2z-z^2|^1_0=1$

**Comment by LeoCosta02** [comment]
@Elia-Belli nel punto uno come fanno a venire quei due integrali da 0 a z e da z a 2?
